asv/template/asv.conf.json

```diff
@@ -36,6 +36,13 @@
     // environments in.  If not provided, defaults to "env"
     // "env_dir": "env",
 
+    // The tool to use to create environments.  May be "conda",
+    // "virtualenv" or other value depending on the plugins in use.
+    // If missing or the empty string, the tool will be automatically
+    // determined by looking for tools on the PATH environment
+    // variable.
+    // "environment_type": "",
```

Comment:
The new "environment_type" option doesn't show up in the reference docs ... I think you have to add it in `docs/source/asv.conf.json.rst`, no?

An unrelated point is that there's this warning:
```
/Users/deil/code/asv/docs/source/asv.conf.json.rst:66: WARNING: Inline literal start-string without end-string.
```
Maybe run `build_sphinx` on travis-ci or if it already does make warnings fail the build?